exchanges/deribit/deribit_test.go

```diff
@@ -3266,11 +3268,79 @@ func setupWs() {
 	}
 }
 
-func TestGenerateDefaultSubscriptions(t *testing.T) {
+func TestGenerateSubscriptions(t *testing.T) {
```

Comment:
This is erroring: [here](https://github.com/thrasher-corp/gocryptotrader/actions/runs/12288877765/job/34293412108?pr=1636#step:10:541)

Reply:
Fixed https://github.com/gbjk/gocryptotrader/commit/c3e5308ae

Was the result of the upstream fix to master to respect disabled websocket assets, and the test didn't know about that.